x-pack/plugins/cloud_security_posture/server/routes/benchmarks/benchmarks.ts

```diff
@@ -101,17 +105,23 @@ export const defineGetBenchmarksRoute = (router: CspRouter): void =>
       const cspContext = await context.csp;
 
       try {
-        const cspPackagePolicies = await getCspPackagePolicies(
+        const packagePolicies = await getCspPackagePolicies(
           cspContext.soClient,
           cspContext.packagePolicyService,
           CLOUD_SECURITY_POSTURE_PACKAGE_NAME,
           request.query,
           POSTURE_TYPE_ALL
         );
 
+        const cspPackagePolicies = packagePolicies.items.filter(
+          (item: PackagePolicy) =>
+            item.vars?.type?.value !== VULN_MGMT_POLICY_TEMPLATE &&
+            item.vars?.type?.value !== CNVM_POLICY_TEMPLATE
```

Comment:
why are there 2 options? do we actually save this one integration under two different names?

Reply:
Currently, we save the `item.vars?.type?.value` under VULN_MGMT_POLICY_TEMPLATE. I added the additional check in case the item.vars?.type?.value  changes to `CNVM_POLICY_TEMPLATE`